examples/framework-preact/src/components/Counter.tsx

```diff
@@ -1,6 +1,10 @@
 import { h, Fragment } from 'preact';
+import { lazy, Suspense } from 'preact/compat';
 import './Counter.css';
 
+const Message = lazy(async () => import('./Message'));
+const Fallback = () => <p>i'm loader, which nobody should see</p>;
```

Comment:
Since this is a public template that people use, perhaps you can word this a bit nicely:
```suggestion
const Fallback = () => <p>Loading...</p>;
```